### PR TITLE
TRACEFOSS-893 [UUX][FE] Rework user menu to align with CX guidelines

### PR DESCRIPTION
**Pull request by ds-ext-dskrzypczak**
TRACEFOSS-893

### CODE REVIEW

File: src/app/modules/core/layout/header/user-navigation/user-menu.component.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <div class="user-menu-content">
+    <div (click)="this.expand($event)">
+      <button class="user-menu-btn" data-testid="user-menu" id="user-menu" aria-haspopup="true">
+        <span class="sr-only">{{ 'layout.nav.openUserMenu' | i18n }}</span>
+        <app-avatar></app-avatar>
+      </button>
+    </div>
+    <div
+      *ngIf="this.isExpanded"
+      [class.fade-in]="this.isExpanded"
```
Comment (evia-fbedarf) [inline src/app/modules/core/layout/header/user-navigation/user-menu.component.html:33]: you don't need the "this" prefix in HTML

File: src/app/modules/shared/components/avatar/avatar.component.ts [hunk 2]
```diff
@@ -28,5 +28,5 @@ import { Component, Input } from '@angular/core';
 })
 export class AvatarComponent {
   @Input() name = '';
-  @Input() imageUrl: string;
+  @Input() imageUrl: string = '';
```
Comment (evia-fbedarf) [inline src/app/modules/shared/components/avatar/avatar.component.ts:31]: We don't need this anymore, right? Could you remove the image from our assets folder as well.

**Review by evia-fbedarf — CHANGES_REQUESTED** [review]

File: src/app/modules/core/layout/header/user-navigation/user-menu.component.html (hunk 1 above)
Comment (ds-ext-dskrzypczak) [reply src/app/modules/core/layout/header/user-navigation/user-menu.component.html:33]: :+1:

**Review by evia-fbedarf — DISMISSED** [review]
LGTM

**Review by evia-fbedarf — APPROVED** [review]
LGTM

**Pull request merged**